@ndlib/gatsby-theme-marble/src/components/Common/style.module.css

```diff
@@ -0,0 +1,90 @@
+
```

Comment:
We are actively trying to remediate `style.module.css` files out of the project since they don't play well with the theme layer. These should be rewritten as `sx.js` files and the component using them should use `jsx` from 'theme-ui`.